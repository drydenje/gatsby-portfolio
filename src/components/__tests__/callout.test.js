import React from 'react';
import renderer from 'react-test-renderer';

import Callout from '../callout';
import Headshot from '../images/Square.jpg';

describe('Component - Callout', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Callout />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a section title to the Callout component', () => {
    const tree = renderer.create(<Callout headingText="Portfolio" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a picture to the Callout component', () => {
    const tree = renderer.create(<Callout headshot={Headshot} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
