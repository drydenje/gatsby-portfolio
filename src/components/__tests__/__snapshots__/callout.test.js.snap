// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Callout renders a picture to the Callout component 1`] = `
<div
  className="sc-bdVaJa bBuFaz"
>
  <img
    className="sc-htpNat qHGID"
    src="test-file-stub"
  />
</div>
`;

exports[`Component - Callout renders a section title to the Callout component 1`] = `
<div
  className="sc-bdVaJa bBuFaz"
>
  <h1
    className="sc-bwzfXH fSkwLX"
  >
    Portfolio
  </h1>
</div>
`;

exports[`Component - Callout renders correctly 1`] = `
<div
  className="sc-bdVaJa bBuFaz"
/>
`;
